apps/project/tasks.py

```diff
@@ -25,3 +25,20 @@ def process_project_task(project_id: int):
         logger.error("Project geometry load failed", exc_info=True)
         project.update_status(Project.Status.FAILED, True)
         return False
+
+
+@shared_task
+def push_project_to_firebase(project_id: int):
+    with CeleryLock.redis_lock(CeleryLock.Key.PUSH_PROJECT_TO_FIREBASE.format(project_id)) as acquired:
+        if not acquired:
+            logger.warning("Project(id: %s) push project to firebase already running", project_id)
+            return None
+
+    project = Project.objects.get(pk=project_id)
+    try:
+        project_type_handler = get_project_type_handler(project.project_type_enum)(project)
+        project_type_handler.push_to_firebase()
+        return True
+    except Exception:
+        logger.error("Push project to firebase failed", exc_info=True)
+        return False
```

Comment:
We can now remove the try/except from here as it is already handled by push_to_firebase

Reply:
Done!